st2actions/st2actions/runners/mistral/v2.py

```diff
@@ -170,8 +170,15 @@ def run(self, action_parameters):
                                                        **options)
 
         status = ACTIONEXEC_STATUS_RUNNING
-        query_context = {'mistral_execution_id': str(execution.id)}
-        LOG.info('Mistral query_context is %s' % query_context)
         partial_results = {'tasks': []}
 
-        return (status, partial_results, query_context)
+        context = {
```

Comment:
It is irrelevant per the rest of the comments. I was trying to see how to limit scope of this PR.

A single commit often times makes it hard to follow reasoning. Also, part of your comments helped me at least understand your reasoning.